chart/compass/charts/async-operations/templates/crds/crds.yaml

```diff
@@ -0,0 +1,120 @@
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: OperationSpec defines the desired state of Operation
+          properties:
+            correlation_id:
+              type: string
+            operation_category:
+              type: string
+            operation_id:
+              type: string
+            operation_type:
+              enum:
+              - Create
+              - Update
+              - Delete
+              type: string
+            request_data:
+              type: string
+            resource_id:
+              type: string
+            resource_type:
+              type: string
+            webhook_ids:
+              items:
+                type: string
+              type: array
+          required:
+          - correlation_id
+          - operation_category
+          - operation_id
+          - operation_type
+          - resource_id
+          - resource_type
+          type: object
+        status:
+          description: OperationStatus defines the observed state of Operation
+          properties:
+            message:
+              type: string
+            status:
+              enum:
+              - Ready
+              - InProgress
+              - Failed
+              type: string
+            webhooks:
+              items:
+                properties:
+                  last_poll_timestamp:
+                    type: string
+                  retries_count:
+                    format: int32
+                    type: integer
+                  state:
+                    enum:
+                    - Success
+                    - Failed
+                    - Polling
+                    type: string
+                  webhook_id:
+                    type: string
+                  webhook_poll_url:
+                    type: string
+                required:
+                - last_poll_timestamp
+                - retries_count
+                - state
+                - webhook_id
+                - webhook_poll_url
+                type: object
+              type: array
+          required:
+          - status
+          type: object
+      type: object
+  version: v0
+  versions:
+  - name: v0
+    served: true
+    storage: true
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
```

Comment:
The status key is not modifiable by hand from a user. If you're looking the `kubectl get operation` and `kubectl get operations`, they're already available, it can be seen from [here](https://github.com/kyma-incubator/compass/blob/c6f8f3250df86f58452a1a6eac275be03a4ce862/chart/compass/charts/async-operations/templates/crds/crds.yaml#L20).